classes/form/dataflow_form.php

```diff
@@ -71,6 +71,14 @@ public function definition() {
             get_string('field_config', 'tool_dataflows'),
             ['cols' => 50, 'rows' => 7]
         );
+        $outputsexample['reference'] = \html_writer::tag('code',  htmlentities('${{dataflows.config.<name>}}'));
+        $examplestring = <<<'EOT'
+data:
+  configvar: 12     # Accessed as ${{dataflows.config.data.configvar}}
+EOT;
+        $outputsexample['example'] = \html_writer::tag('pre', $examplestring);
+        $mform->addElement('static', 'config_help', '',
+                get_string('field_config_help', 'tool_dataflows', $outputsexample));
```

Comment:
So I'm not sure what's happening here, but are you able to use 4 spaces for indentation?

```suggestion
            get_string('field_config_help', 'tool_dataflows', $outputsexample));
```

Reply:
Potentially it might be a case where setting this on a single line is fine also?